examples/06_Combining_Models.ipynb

```diff
@@ -250,6 +250,247 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's create an EnsembleModel which combines each of these."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from progpy import EnsembleModel\n",
+    "m_ensemble = EnsembleModel(\n",
+    "    models=(m_circuit, m_circuit_2, m_circuit_3))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's evaluate the performance of the combined model using real battery data from NASA's prognostic data repository. See 07. Datasets for more detail on accessing data from this repository"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from progpy.datasets import nasa_battery\n",
+    "data = nasa_battery.load_data(batt_id=8)[1]\n",
+    "RUN_ID = 0\n",
+    "test_input = [{'i': i} for i in data[RUN_ID]['current']]\n",
+    "test_time = data[RUN_ID]['relativeTime']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To evaluate the model we first create a future loading function that uses the loading from the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def future_loading(t, x=None):\n",
+    "    for i, mission_time in enumerate(test_time):\n",
+    "        if mission_time > t:\n",
+    "            return m_circuit.InputContainer(test_input[i])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we will simulate the ensemble model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "t_end = test_time.iloc[-1]\n",
+    "results_ensemble = m_ensemble.simulate_to(t_end, future_loading)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, we compare the voltage predicted by the ensemble model with the ground truth from dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from matplotlib import pyplot as plt\n",
+    "fig = plt.plot(test_time, data[RUN_ID]['voltage'], color='green', label='ground truth')\n",
+    "fig = plt.plot(results_ensemble.times, [z['v'] for z in results_ensemble.outputs], color='red', label='ensemble')\n",
+    "plt.xlabel('Time (s)')\n",
+    "plt.ylabel('Voltage')\n",
+    "plt.legend()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The ensemble model actually performs pretty poorly here. This is mostly because there's an outlier model (m_circuit_3). This can resolved using a different aggregation method. Let's update the ensemble model to use median and resimulate"
```

Comment:
```suggestion
    "The ensemble model actually performs pretty poorly here. This is mostly because there's an outlier model (m_circuit_3). This can be resolved using a different aggregation method. Let's update the ensemble model to use median and resimulate"
```